javascript/src/components/navigation/HelpMenu.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {PropTypes} from 'react';
+import { NavDropdown, MenuItem } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+
+import Routes from 'routing/Routes';
+
+const HelpMenu = React.createClass({
+  propTypes: {
+    active: PropTypes.bool.isRequired,
+  },
+  render() {
+    return (
+      <NavDropdown navItem title="Help" id="user-menu-dropdown" active={this.props.active}>
+        <LinkContainer to={Routes.GETTING_STARTED}>
+          <MenuItem>Getting Started</MenuItem>
+        </LinkContainer>
+        <MenuItem href="http://docs.graylog.org" target="blank">
+          <i className="fa fa-external-link"></i>
+          &nbsp;
+          Documentation
+        </MenuItem>
+        <MenuItem divider/>
+        <MenuItem href="https://www.graylog.org/support/" target="blank">
+          <i className="fa fa-external-link"></i>
+          &nbsp;
+          Get Support
+        </MenuItem>
+              <MenuItem href="https://graylog.wufoo.com/forms/qup3ebj0kp9cfo/" target="blank">
```

Comment:
Are those links stable? Otherwise we should probably create a redirect URL for which we can adapt the target if necessary. Same for the "Ask a Question" menu item.